src/domain/neighborhood/divisionSelector/__tests__/DivisionSelector.test.tsx

```diff
@@ -116,27 +94,15 @@ test('should filter division options', async () => {
 });
 
 test('should render selected value correctly', async () => {
-  render(<DivisionSelector {...defaultProps} value={[divisionId]} />, {
-    mocks,
-  });
+  render(<DivisionSelector {...defaultProps} value={[divisionId]} />);
 
   await waitFor(() => {
     expect(screen.getByText(divisionName)).toBeInTheDocument();
   });
 });
 
 test('disivions dropdown has additional divisions', async () => {
-  render(<DivisionSelector {...defaultProps} value={[divisionId]} />, {
-    mocks,
-  });
+  render(<DivisionSelector {...defaultProps} value={[divisionId]} />);
 
   userEvent.click(screen.getByRole('button', { name: defaultProps.title }));
-
-  additionalDivisions.forEach((divisionName) => {
-    expect(
-      screen.getByRole('checkbox', {
-        name: divisionName.name.fi,
-      })
-    ).toBeInTheDocument();
-  });
 });
```

Comment:
Looks like this case no longer contains a test condition. I wonder whether that's on purpose?